web/_includes/icon.html

```diff
@@ -23,6 +23,8 @@
     <span class="fas fa-fw fa-project-diagram color-white"></span>
   {% when 'button-email' %}
     <span class="fas fa-fw fa-envelope color-white"></span>
+  {% when 'unused-openssl-icon' %}
+    <span class="fas fa-times {{ include.float }} color-verylightgray"></span>
```

Comment:
Please use a fixed-width icon to be consistent with the style (adding the class `fa-fw`).

Reply:
Secondly, add a hover title text with the explanation (see the library icons).